tcms/xmlrpc/tests/test_testcase.py

```diff
@@ -1 +1,47 @@
 # -*- coding: utf-8 -*-
+
+from django import test
+
+from tcms.xmlrpc.api import testcase as XmlrpcTestCase
+from tcms.xmlrpc.tests.utils import make_http_request
+from tcms.xmlrpc.tests.utils import user_should_have_perm
+
+from tcms.tests.factories import TestCaseFactory
+from tcms.tests.factories import UserFactory
+
+__all__ = (
+    'TestNotificationRemoveCC',
+)
+
+
+class TestNotificationRemoveCC(test.TestCase):
+    ''' Tests the XMLRPM testcase.notication_remove_cc method '''
```

Comment:
You have used single quotes for doc strings far more often than double quotes. I merely looked at what was already there. So which one style do you prefer so I can open an issue to update the other one ?